packages/bsync/src/purchases/addPurchaseOperation.ts

```diff
@@ -7,11 +7,13 @@ export const addPurchaseOperation = async (
   actorDid: string,
   entitlements: string[],
 ) => {
+  const sortedEntitlements = [...entitlements].sort()
```

Comment:
May want to mention here that they're sorted for later comparison. I can imagine someone in the future deleting this without knowing the purpose haha.